bin/s2nc.c

```diff
@@ -236,6 +240,10 @@ int main(int argc, char *const *argv)
     int echo_input = 0;
     int use_corked_io = 0;
     int use_tls13 = 0;
+    int keyshares_count = 0;
+    char keyshares[S2N_ECC_EVP_SUPPORTED_CURVES_COUNT][10];
```

Comment:
nit: s/10/S2N_MAX_ECC_CURVE_NAME_LENGTH ?

Reply:
I could add a constant for this value.